components/webdriver_server/lib.rs

```diff
@@ -970,6 +970,13 @@ impl Handler {
         }
     }
 
+    fn handle_delete_cookies(&self) -> WebDriverResult<WebDriverResponse> {
+        let (sender,_) = ipc::channel().unwrap();
```

Comment:
We should wait for the response from the receiver that's created here and return any error that occurs.